src/cf-forms/src/cf-forms.less

```diff
@@ -219,6 +219,663 @@ textarea {
+                               type="checkbox"
+                               name="form_id"
+                               id="input_id_dogs">
+                        <label class="cf-input_label"
+                               for="input_id_dogs">
+                            Dogs
+                        </label>
+                    </div>
+                </div>
+            </fieldset>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            fieldset
+                .form-l_col-1
+                    legend
+                .form-l-inset
+                    .form-l-inset_container
+                        input[type="checkbox"].cf-input
+                        label.cf-input_label
+        notes:
+          - "Use the markup structure above to add checkbox .cf-input elements."
+      - name: Radio buttons
+        markup: |
+            <fieldset class="u-reset">
+                <div class="form-l_col
+                            form-l_col-1">
+                    <legend class="form-label-header">
+                        Pick an option
+                    </legend>
+                </div>
+                <div class="form-l_col
+                            form-l_col-1-2
+                            form-l-inset">
+                    <div class="form-l-inset_container">
+                        <input class="cf-input"
+                               type="radio"
+                               name="form_id"
+                               id="input_id_1">
+                        <label class="cf-input_label"
+                               for="input_id_1">
+                            Yes
+                        </label>
+                    </div>
+                </div>
+                <div class="form-l_col
+                            form-l_col-1-2
+                            form-l-inset">
+                    <div class="form-l-inset_container">
+                        <input class="cf-input"
+                               type="radio"
+                               name="form_id"
+                               id="input_id_2">
+                        <label class="cf-input_label"
+                               for="input_id_2">
+                            No
+                        </label>
+                    </div>
+                </div>
+            </fieldset>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            fieldset
+                .form-l_col-1
+                    legend
+                .form-l-inset
+                    .form-l-inset_container
+                        input[type="radio"].cf-input
+                        label.cf-input_label
+        notes:
+          - "Use the markup structure above to add radio button .cf-input elements."
+    tags:
+      - cf-forms
+*/
+
+@input-active:          #0072ce; // pacific.
```

Comment:
So there's no canonical, central source that defines those colors people use? Is there a `cf-theme-overrides.less` in Capital Framework where I can add those? Otherwise people won't know which colors to add to their projects.